content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -141,6 +141,80 @@ limit.
 On Linux, resource allocations for Pod level control groups (cgroups) are based on the effective Pod
 request and limit, the same as the scheduler.
 
+### Adopting sidecar containers
+
+This section is relevant for people adopting sidecar containers for their workloads.
+
+The sidecar containers feature is in beta state starting Kubernetes version 1.29 and is enabled by default.
```

Comment:
Better use the shortcode for feature gates.
This sentence becomes invalid when the gate is removed.
By the way, "sidecar containers" is not the name of the gate.